praw/models/reddit/subreddit.py

```diff
@@ -2707,9 +2709,23 @@ def upload_banner_additional_image(self, image_path):
            reddit.subreddit('SUBREDDIT').stylesheet.upload_banner_additional_image('banner.png')
 
         """
+        alignment = {}
+        if align is not None:
+            if align not in {"left", "center", "right"}:
+                raise ValueError(
+                    "align argument must be either "
+                    "`left`, `center`, or `right`"
+                )
+            if align == "center":
```

Comment:
Why make this correction instead of simply allowing `centered`?

Reply:
Because I’m a fan of the [tabulate module] and those values would *align* with tabulate’s `colalign` parameter.

[tabulate module]: https://bitbucket.org/astanin/python-tabulate/src/master/

Reply:
Also, CSS uses “center” for `text-align`, `content-align`, `box-align`, etc.

Reply:
I'd greatly prefer if we stick with Reddit's conventions for naming to make it easier to switch between PRAW, raw requests to Reddit, and/or other libraries. This has the added benefit of keeping the code a bit simpler.

Reply:
Okay.